src/plugins/api-gateway/cli/deploy-apis.js

```diff
@@ -1,19 +1,121 @@
 'use strict';
 
+// Nice ES6 syntax
+// const { Promise, _, icli } = require('@lager/lager/lib/lager').import;
 const lager = require('@lager/lager/lib/lager');
+const Promise = lager.import.Promise;
+const _ = lager.import._;
+const icli = lager.import.icli;
 
-module.exports = function deployApiCmd(program, inquirer) {
-  return program
-  .command('deploy-apis')
-  .description('deploy apis')
-  .option('-r, --region [region]', 'select the AWS region', process.env.AWS_DEFAULT_REGION || 'us-east-1')
-  .option('-s, --stage [stage]', 'select the API stage', process.env.API_GATEWAY_STAGE || 'v0')
-  .option('-e, --environment [environment]', 'select the environment', process.env.NODE_ENV || 'NO_ENV')
-  .action(options => {
-    lager.getPlugin('api-gateway').deploy(
-      options.region,
-      options.stage,
-      options.environment
-    );
+const plugin = lager.getPlugin('api-gateway');
+
+/**
+ * This module exports a function that enrich the interactive command line and return a promise
+ * @return {Promise} - a promise that resolve when the operation is done
+ */
+module.exports = () => {
+  // Build the list of available APIs andAWS regions for input verification and interactive selection
+  const choicesLists = getChoices()
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["choicesLists" is defined but never used](https://www.codacy.com/app/lager/lager/file/2780163937/issues/source?bid=3360491&fileBranchId=3365294#l18)